### ISSUE TITLE
Cobalt vocal and instrument separator

### ISSUE THREAD

**Issue by KyleeC137** [imputnet/cobalt#1021]
### describe the feature you'd like to see

For the audio purposes, downloading a piece from the web most often could benefit of the ability of separating the vocals from the background music.

https://github.com/Anjok07/ultimatevocalremovergui

heres a repo that could be used on this behalf. 

### additional context

I dont know much about code, only the fact that there is available code out there that could perhaps be put together in a way to benefit the final 'product'

**Comment by lostdusty** [comment]
I think thats out of cobalt's scope

**Comment by KyleeC137** [comment]
> I think thats out of cobalt's scope

If it is, then there's no problem at all.
Thank you for the feedback, tamo junto dusty ❤️

**Comment by KwiatekMiki** [comment]
cobalt is only for downloading

**Comment by elouangrimm** [comment]
> cobalt is only for downloading

That is not true, cobalt has the remux too. This splitter would work fine if it was added as a separate tool along the sidebar, as I love the style that cobalt has and I hate the UI of every other splitter. This would be a great feature.
